releases/v19.1.0.md

```diff
@@ -0,0 +1,129 @@
+
+- The normalized results of certain timestamp + duration operations involving year or month durations have been adjusted to agree with the values returned by PostgreSQL.
+
+- The `CHANGEFEED` [`experimental-avro` option](../v19.1/create-changefeed.html#options) has been renamed `experimental_avro`.
+
+- Timezone abbreviations, such as `EST`, are no longer allowed when parsing or converting to a date/time type. Previously, an abbreviation would be accepted if it were an alias for the session's timezone.
+
+- The way composite [foreign key](../v19.1/foreign-key.html) matches are evaluated has changed to match the Postgres behavior. If your schema currently uses composite keys, it may require updates, since this change may affect your foreign key constraints and cascading behavior. For more details and guidance, see [this note](v2.2.0-alpha.20190114.html#composite-foreign-key-matching).
+
+- Mutation statements like [`UPDATE`](../v19.1/update.html) and [`INSERT`](../v19.1/insert.html) no longer attempt to guarantee mutation or output ordering when an `ORDER BY` clause is present. It is now an error to use `ORDER BY` without `LIMIT` with the `UPDATE` statement.
+
+### Known limitations
+
+For information about limitations we've identified in CockroachDB v19.1, with suggested workarounds where applicable, see [Known Limitations](../v19.1/known-limitations.html).
+
+### Documentation
+
+Topic | Description
+------|------------
+**Security** | Added an [overview of CockroachDB security](../v19.1/security-overview.html), with a dedicated page on [authentication](../v19.1/authentication.html), [encryption](../v19.1/encryption.html), [authorization](../v19.1/authorization.html), and [SQL audit logging](../v19.1/sql-audit-logging.html).
+**Troubleshooting** | Added much more guidance on [troubleshooting cluster setup](../v19.1/cluster-setup-troubleshooting.html) and [troubleshooting SQL behavior](../v19.1/query-behavior-troubleshooting.html).
+**Architecture** | Added the [Life of a Distributed Transaction](../v19.1/architecture/life-of-a-distributed-transaction.html), which details the path that a query takes through CockroachDB's architecture, starting with a SQL client and progressing all the way to RocksDB (and then back out again). Also added [Reads and Writes in CockroachDB](../v19.1/architecture/reads-and-writes-overview.html), which explains how reads and writes are affected by the replicated and distributed nature of data in CockroachDB.
+**Production Guidance** | Expanded the [Production Checklist](../v19.1/recommended-production-settings.html) with more current hardware recommendations and additional guidance on storage, file systems, and clock synch. Also added a library of common [Cluster Topology Patterns](../v19.1/cluster-topology-patterns.html).
```

Comment:
Nit: "Synch" caught my eye while reading but it's not wrong, just the less common variant according to [Merriam-Webster](https://www.merriam-webster.com/dictionary/sync).  I'd vote "sync" personally.

Reply:
Good catch, but I'll just changed it to `synchronization`.